test/functional/p2p-segwit.py

```diff
@@ -1905,7 +1905,7 @@ def run_test(self):
         self.test_unnecessary_witness_before_segwit_activation()
         self.test_witness_tx_relay_before_segwit_activation()
         self.test_block_relay(segwit_activated=False)
-        self.test_p2sh_witness(segwit_activated=False)
+        #self.test_p2sh_witness(segwit_activated=False)
```

Comment:
In commit "[qa] Remove some pre-activation segwit tests"

Should actually remove